examples/first-component/src/components/Scientist/Scientist.tsx

```diff
@@ -0,0 +1,47 @@
+// https://react.dev/learn/passing-props-to-a-component#challenges
```

Comment:
Please help update the naming of this file into `index.tsx`

Reply:
Same as ScientistList

Reply:
Fixed